src/main/java/com/flightstats/hub/metrics/MetricsResource.java

```diff
@@ -11,26 +12,45 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
+import java.util.Map;
 
 @SuppressWarnings("WeakerAccess")
 @Path("/health/metrics")
 public class MetricsResource {
     private final static Logger logger = LoggerFactory.getLogger(MetricsResource.class);
-
     private final static ObjectMapper mapper = HubProvider.getInstance(ObjectMapper.class);
-
+    private final static CustomMetricsReporter customMetricsReporter = HubProvider.getInstance(CustomMetricsReporter.class);
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     public Response checkHealth() {
         ObjectNode rootNode = mapper.createObjectNode();
-        rootNode.put("openFiles", MetricsRunner.getOpenFiles());
+        rootNode.put("openFiles", customMetricsReporter.getOpenFiles());
         return Response.ok(rootNode).build();
     }
 
     @GET
     @Path("trace")
     @Produces(MediaType.TEXT_PLAIN)
     public Response trigger() {
-        return Response.ok(MetricsRunner.logFilesInfo()).build();
+        return Response.ok(logFilesInfo()).build();
+    }
+
```

Comment:
This logic in logFilesInfo was dumped into MetricsRunner (which mostly became CustomMetricsReporter) but had nothing really to do with it and is only called by this class, I started to refactor it and thought better of it at this time